include/roboteam_ai/world/Field.h

```diff
@@ -107,18 +121,32 @@ class Field {
     // The center y-coordinate of the field (the y-coordinate that corresponds with the center of the field)
     std::optional<double> centerY;
 
-    // The leftmost x-coordinate of the field (the x-coordinate closest to our goal)
+    // The leftmost x-coordinate of the field which is the lowest x-coordinate value (is a negative value) and is the x-coordinate closest to our goal.
     std::optional<double> leftmostX;
```

Comment:
The comment is useless and vague as well. What is the x-coordinate closest to our goal? the x coordinate of what? what is wrong with field->getLeftFieldLine.x? which is more descriptive

Reply:
No, field->getLeftFieldLine.x is less readable than getting the leftMostX of the field. What I often see happening in the AI is that a vector is asked when only part of the vector is needed and then complicated things are done with that vector to return a part of the vector.

Reply:
but what X? field->getleftfieldline.x clearly shows that it is the xposition of the left field line. in your case, what does it mean? you say it is the leftmost x. But you can go further than the fieldline, which is more left. This is a confusing name and confusing documentation.

Reply:
I give it a more clear name and add more comments which explains it